docs/en/deploy/Running-Alluxio-On-Kubernetes.md

```diff
@@ -1471,11 +1472,14 @@ spec:
 
 #### CSI
 Other than using Alluxio FUSE daemon, you could also use CSI to mount the Alluxio FileSystem into application containers.
+The pods running Alluxio Fuse processes will only be launched when the application pods ask for data from Alluxio through CSI.
+A Fuse pod will be terminated when no pod utilizes the Fuse pod. In other words, the Fuse process exists only when it's needed,
```

Comment:
when no application pod requests data through the Fuse path? Fuse path or Fuse mounted volume or how to describe this part?

Reply:
when it's needed or when it's being used?